dashboard/src/components/BaseCharts/core.tsx

```
@@ -0,0 +1,256 @@
+    instance: {
+      on: (
+        arg0: string,
+        arg1: string | Record<string, any> | undefined,
+        arg2: (param: any) => void,
+      ) => void;
+    },
+    events: EChartsReactProps['onEvents'],
+  ) {
+    function _bindEvent(eventName: string, func: Function, query?: string | Record<string, any>) {
+      // ignore the event config which not satisfy
+      if (isString(eventName) && isFunction(func)) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)